cli/spark/aztklib.py

```diff
@@ -0,0 +1,23 @@
+import cli.config as config
+import aztk.spark
+
+
+def load_spark_client():
+    secrets_config = config.SecretsConfig()
```

Comment:
just add a small doc here saying this method create the client for you using the local user environment